libgnucash/engine/engine-utilities.scm

```diff
@@ -93,33 +93,31 @@
   (hashx-set! account-hash account-assoc table account value))
 
 ;; Splits
-(export split-same?)
-(export split-in-list?)
-
+(export split-same?)                    ;deprecated
+(export split-in-list?)                 ;deprecated
 (define (split-same? s1 s2)
+  (issue-deprecation-warning "split-same? is deprecated.")
```

Comment:
On a general note: it would be *very* helpful to function consumers if your deprecation warnings hinted at the alternative.
So make the message something like

`"split-same? is deprecated. Please use xyz instead.`

While no doubt the replacement is blindingly obvious to you, it certainly isn't for me :)
And we want our code to be conductive for newcomers also...